core/src/main/scala/org/apache/spark/sql/delta/commands/ConvertToDeltaCommand.scala

```diff
@@ -266,16 +269,30 @@ abstract class ConvertToDeltaCommandBase(
    * Given the file manifest, create corresponding AddFile actions for the entire list of files.
    */
   protected def createDeltaActions(
+      sparkSession: SparkSession,
       manifest: ConvertTargetFileManifest,
       partitionSchema: StructType,
       txn: OptimisticTransaction,
-      fs: FileSystem): Iterator[AddFile] = {
+      fs: FileSystem,
+      collectStats: Boolean): Iterator[AddFile] = {
+    val initialSnapshot = new InitialSnapshot(txn.deltaLog.dataPath, txn.deltaLog, txn.metadata)
+    val collectStatsDeltaConfiguration = sparkSession.sessionState.conf
```

Comment:
you don't need `sparkSession`. you can just do `conf.getConf`

also let's name this variable `statsEnabled`